### ISSUE TITLE
Change the project file save dialog: too wordy and some of the logic seems wrong

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#173]
```
For example, after a file has been saved, but without a project file, the 
dialog comes back and prompts for save file as. This is superfluous, once the 
file is saved (with or without a project file) one should be good to go. While 
it is a good idea to have a project file, personally I work without one most of 
the time.

```

Original issue reported on code.google.com by `lixun...@gmail.com` on 30 Jun 2015 at 3:43

**Comment by jkoschinsky** [comment]
Verified in GeoDa 1.7.63
